README.md

```diff
@@ -25,6 +25,12 @@ The package can be installed using the AEM Package Manager or directly from the
 mvn -PautoInstallPackage install
 ```
 
+## AEM6.x
```

Comment:
What about making this headline say "Oak" and some more information about this Oak Index extension like. "The oakindex-package contains an optimized Oak index to cover all queries being issued by the Access Control Tool. To build (and optionally deploy) the content-package use the Maven profile `oakindex`. This package is only compatible with Oak and even there it is optional (as it will only speed up queries).